awx/ui_next/src/screens/Template/TemplateList/TemplateListItem.jsx

```diff
@@ -31,6 +31,12 @@ function TemplateListItem({ i18n, template, isSelected, onSelect, detailUrl }) {
       (!template.summary_fields.inventory &&
         !template.ask_inventory_on_launch));
 
+  const location = useLocation();
+
+  if (location.pathname.startsWith('/projects')) {
+    detailUrl = `/templates/job_template/${template.id}/details`;
```

Comment:
I don't think a lower-level list item component should be smart enough to introspect what view it is on.

Instead of inspecting the URL, could you instead pass a different `detailUrl` from the parent project component

Reply:
edit: To to this, we'll first need to do: https://github.com/ansible/awx/pull/6015#issuecomment-589298424

Let's make that a part of this PR.

Reply:
@jakemcdermott We do have similar behavior in Inventory Host where we borrow the Host list, and form and use it within the Inventory context.  Do you think that should be redone as well?

Reply:
No. Let's fix that later and not in this PR.